pkg/cloud/cloud.go

```diff
@@ -56,6 +56,10 @@ var (
 		VolumeTypeST1,
 		VolumeTypeStandard,
 	}
+
+	volumeModificationDuration   = 1 * time.Second
```

Comment:
This doesn't match up with what went into the main branch. Is there a reason for the difference? This seems cleaner than the hard coding in the main branch.

Reply:
Example: https://github.com/openshift/aws-ebs-csi-driver/blob/master/pkg/cloud/cloud.go#L962-L969

Reply:
Thank you for noticing! Logically code in this branch and main branch is same, but code in this branch is direct cherry-pick of https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/552 whereas code in main branch is bit "unclean", because we went through several iterations while debugging this and it took lot of time to arrive on final code. 

At some point once main branch gets rebased against upstream, it will have identical code but I am not too fussed about that right now because logically they are same.